<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>动态编码仿真</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="zx_recieve">
       <property name="font">
        <font>
         <family>等线</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(147,112,219);

color:white;</string>
       </property>
       <property name="text">
        <string>正线接车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cx_recieve">
       <property name="font">
        <font>
         <family>等线</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(138,43,226);

color:white;</string>
       </property>
       <property name="text">
        <string>侧线接车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lg_recieve">
       <property name="font">
        <font>
         <family>等线</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(153,50,204);

color:white;</string>
       </property>
       <property name="text">
        <string>18号接车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zx_send">
       <property name="font">
        <font>
         <family>等线</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(147,112,219);

color:white;</string>
       </property>
       <property name="text">
        <string>正线发车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cx_send">
       <property name="font">
        <font>
         <family>等线</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(138,43,226);

color:white;</string>
       </property>
       <property name="text">
        <string>侧线发车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lg_send">
       <property name="font">
        <font>
         <family>等线</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(153,50,204);

color:white;</string>
       </property>
       <property name="text">
        <string>18号发车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_btn">
       <property name="font">
        <font>
         <family>Bahnschrift SemiBold</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(178,34,34);

color:white;</string>
       </property>
       <property name="text">
        <string>RUN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_btn">
       <property name="font">
        <font>
         <family>Bahnschrift SemiBold</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(102,139,139);

color:white;</string>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
